### PR TITLE
Use of pointer to an array

### PR DESCRIPTION
**Pull request by YuweAEC**
image

### CODE REVIEW

File: daily-code-update/Array/array7.c [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#include<stdio.h>
+#include<conio.h>
+
+int pointerArray();
```
Comment (gauravsinghaec) [inline daily-code-update/Array/array7.c]: why the return type is int?

File: daily-code-update/Array/array7.c [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+int main ()
```
Comment (gauravsinghaec) [inline daily-code-update/Array/array7.c:6]: same here, return type?

**Comment by gauravsinghaec** [comment]
Check the PR title. is it good?

**Comment by YuweAEC** [comment]
> Check the PR title. is it good?

Corrected

File: daily-code-update/Array/array7.c (hunk 2 above)
Comment (YuweAEC) [reply daily-code-update/Array/array7.c:6]: corrected

File: daily-code-update/Array/array7.c (hunk 1 above)
Comment (YuweAEC) [reply daily-code-update/Array/array7.c]: corrected

**Pull request merged**